packages/zero-runtime/README.md

```diff
@@ -23,6 +24,14 @@ A zero-runtime CSS-in-JS library that extracts the colocated styles to their own
 
 Zero-runtime supports Next.js and Vite with future support for more bundlers. You must install the corresponding plugin, as shown below.
 
+### Why this project exists?
+
+We first introduced an Emotion-based CSS-in-JS styling engine to Material UI back in the v4 to v5 transition. We wanted to support color transformations, theme variables, and other features that wouldn't be possible without a runtime solution. However, this has notably impacted performance and has been a frequent piece of feedback we've heard ever since.
+
+As CSS keeps rapidly evolving with features such as CSS variables and color-mix, to name a few, there's no practical reason to keep a runtime solution. Additionally, React Server Components entered the mix of functionalities Material UI should support.
+
+We looked around and couldn't find anything that fit precisely, so we started a new bespoke project. {newProductName} is designed to have a smooth migration experience from styled-components and Emotion.
```

Comment:
Actually, we are building on top of [wyw-in-js](https://wyw-in-js.dev/). I think we should give some credit to the project.

Reply:
Note: from my point of view, the reason we choose `wyw-in-js` is because our main goal for v6 is to provide static extraction with the same v5 API as much as possible.

In short, it's like we care very much about the migration experience and result rather than what library we are using.

Reply:
Updated it according to this!